src/strategy.ts

```diff
@@ -1,7 +1,8 @@
 import { Strategy as PassportStrategy } from "passport-strategy";
 import { strict as assert } from "assert";
 import * as url from "url";
-import { Profile, SAML } from ".";
+import { Profile } from ".";
```

Comment:
note that this import is fine: it's a type import, and will be omitted from the transpiled multisamlstrategy.js file